packages/synapse-react-client/src/lib/containers/oauth/CreateOAuthClient.tsx

```diff
@@ -199,246 +213,237 @@ export const CreateOAuthModal: React.FunctionComponent<
-                    style={{ marginTop: '8px' }}
-                    color="error"
-                    variant="headline3"
-                  >
-                    DANGER ZONE
-                  </Typography>
-                  <Typography variant="smallText1">
-                    Editing the following information will render your client
-                    invalid and will require you to create it again and resubmit
-                    verification if needed.
-                  </Typography>
-                  <Row>
-                    <Col lg={6} md={6} sm={12} xs={12}>
-                      <Form.Label htmlFor="redirect-uri-0">
-                        Redirect URI(s)
-                      </Form.Label>
-                      <Tooltip title={uriHelpMessage} placement="top">
-                        <HelpOutlineTwoTone className={`HelpButton`} />
-                      </Tooltip>
+  const sectorIdentifierURIInput = (
+    <StyledFormControl fullWidth variant="standard" margin="normal">
+      <TextField
+        onChange={e => setSectorUri(e.target.value)}
+        placeholder="https://"
+        type="text"
+        value={sectorUri}
+        id="sectorURI"
+        label="Sector Identifier URI"
+        InputLabelProps={{ sx: dangerLabelSx }}
+      />
+    </StyledFormControl>
+  )
 
-                      {redirectUris?.map((singleUri, idx) => (
-                        <div key={idx}>
-                          <Form.Control
-                            id={`redirect-uri-${idx}`}
-                            required
-                            name="uri"
-                            onChange={e => handleUriChange(e, idx)}
-                            value={singleUri.uri}
-                            placeholder="https://"
-                            type="text"
-                          />
-                          {redirectUris.length > 1 && (
-                            <button
-                              onClick={() => handleRedirectUriRemove(idx)}
-                            >
-                              <IconSvg
-                                icon="delete"
-                                sx={{ color: 'error.main' }}
-                              />
-                            </button>
-                          )}
+  const redirectURIInputs = (
+    <StyledFormControl fullWidth variant="standard" margin="normal">
+      {redirectUris?.map((singleUri, idx) => (
+        <div key={idx}>
+          <TextField
+            required={idx === 0}
+            InputLabelProps={{
+              sx: { ...dangerLabelSx, pointerEvents: 'auto', zIndex: 1 },
+            }}
+            label={
+              idx === 0 && (
+                <>
+                  Redirect URI(s)
+                  <Tooltip title={uriHelpMessage} placement="top">
+                    <HelpOutlineTwoTone
+                      className={`HelpButton`}
+                      sx={{ float: 'right' }}
+                    />
```

Comment:
Looking at the screenshots more closely, it looks like this shrank. Any idea why?